README.md

```diff
@@ -1 +1,2 @@
 # wordcount
+Welcome to the wordcount repository
```

Comment:
Hey this looks good, would you add a space between the title and the first sentence.